rcl/src/rcl/node.c

```diff
@@ -166,9 +166,7 @@ rcl_node_init(
   }
   if (validation_result != RMW_NAMESPACE_VALID) {
     const char * msg = rmw_namespace_validation_result_string(validation_result);
-    char fixed_msg[256];
-    rcutils_snprintf(fixed_msg, sizeof(fixed_msg), "%s, result '%d'", msg, validation_result);
-    RCL_SET_ERROR_MSG(fixed_msg, *allocator);
+    RCL_SET_ERROR_MSG_WITH_FORMAT_STRING(*allocator, "%s, result '%d'", msg, validation_result);
```

Comment:
This change fails to compile: (http://ci.ros2.org/job/ci_linux-aarch64/935/consoleFull, http://ci.ros2.org/job/ci_windows/3885/). I didn't have time to investigate why yet

Reply:
fixed, thanks !